doc/guides/general/community_process.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Community Processes
```

Comment:
Is this a copy of the old community processes page? That was replaced with `GOVERNANCE.md` in #21067

Reply:
Yes this is quite outdated, I will update them soon-ish, I just had them ready for testing stuff with

Reply:
Fixed